tests/suites/test_suite_psa_crypto.data

```diff
@@ -6629,6 +6629,13 @@ PSA key derivation: PBKDF2-AES-CMAC-PRF-128, exercise HMAC-SHA-256
 depends_on:PSA_WANT_ALG_PBKDF2_AES_CMAC_PRF_128:PSA_WANT_ALG_CMAC:PSA_WANT_ALG_HMAC:PSA_WANT_ALG_SHA_256:PSA_WANT_KEY_TYPE_AES:PSA_WANT_KEY_TYPE_HMAC
 derive_key_exercise:PSA_ALG_PBKDF2_AES_CMAC_PRF_128:"706173737764":"01":"73616c74":PSA_KEY_TYPE_HMAC:256:PSA_KEY_USAGE_SIGN_HASH:PSA_ALG_HMAC(PSA_ALG_SHA_256)
 
+# Deriving a key on a Weierstrass curve requires a driver call, but we
+# haven't implemented cooked key derivation drivers yet
+# ("key_derivation_output_key" driver entry point). Until then,
+# require built-in support for the curve of the derived key.
+# This limitation does not apply to keys on a Montgomery curve for which
+# derivation doesn't involve potentially-accelerated operations and thus
+# doesn't involve a driver call.
```

Comment:
I think you no longer need this, since #8075.

Requiring `PSA_WANT_KEY_TYPE_ECC_KEY_PAIR_DERIVE` should be enough to enable `ECP_LIGHT` and the built-in version of all required curves. See https://github.com/Mbed-TLS/mbedtls/blob/development/include/mbedtls/config_adjust_legacy_from_psa.h#L187 then https://github.com/Mbed-TLS/mbedtls/blob/development/include/mbedtls/config_adjust_legacy_from_psa.h#L207 and https://github.com/Mbed-TLS/mbedtls/blob/development/include/mbedtls/config_adjust_legacy_from_psa.h#L443